src/internal/pfssync/cache_client.go

```diff
@@ -94,14 +94,14 @@ func (ccfsc *cacheCreateFileSetClient) CopyFile(dst string, src *pfs.File, opts
 		newSrc.Commit = c
 		return errors.EnsureStack(ccfsc.ModifyFile.CopyFile(dst, newSrc, opts...))
 	}
-	id, err := ccfsc.APIClient.GetProjectFileSet(src.Commit.Branch.Repo.Project.GetName(), src.Commit.Branch.Repo.Name, src.Commit.Branch.Name, src.Commit.ID)
+	id, err := ccfsc.APIClient.GetProjectFileSet(src.Commit.Repo.Project.GetName(), src.Commit.Repo.Name, src.Commit.Branch.Name, src.Commit.ID)
```

Comment:
If you’re going to use `GetName` it probably makes sense to use `GetFoo` throughout.